spdk-sys/logwrapper.c

```diff
@@ -4,8 +4,26 @@ void
 maya_log(int level, const char *file, const int line, const char *func,
     const char *format, va_list args)
 {
-    char buf[1024] = {0};
-    int n_written = vsnprintf(buf, sizeof(buf), format, args);
-    logfn(level, file, line, func, &buf[0], n_written);
+    // There is a delicate balance here! This `buf` ideally should not be resized, since a realloc is expensive.
```

Comment:
not sure if the comment is out of date, `realloc` here might imply we're using *alloc/realloc, but looks like we just call malloc down below.